packages/core/src/App/Components/Layout/Header/Components/ToggleMenu/menu-title.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { observer, useStore } from '@deriv/stores';
+import { Icon, Text } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+
+const MenuTitle = observer(() => {
+    const {
+        common: { current_language, is_mobile_language_menu_open, setMobileLanguageMenuOpen },
+    } = useStore();
+    return (
+        <React.Fragment>
+            <div>{localize('Menu')}</div>
```

Comment:
Please use Localize component instead of localize function inside your component! ( this is really important, our components need to be attached to react life cycle, so basically using `localize` in most of places are not quite ok. 
```suggestion
            <div>
                  <Localize i18n_default_text='Menu' />
            </div>
```

Reply:
@niloofar-deriv What's the reason behind it? 🤔 Do they behave differently? 🤔

Reply:
Michio suggested this when we were refactoring the deriv.com's translations

Reply:
@niloofar-deriv The script for extracting strings in deriv-com is different 👀

Reply:
But the rules of React are the same :eyes: 

Anyways we can ignore this comment :pray: